README.md

```diff
@@ -4,39 +4,53 @@
 [![Docker Repository on Quay](https://quay.io/repository/redhat-developer/devconsole-operator/status "Docker Repository on Quay")](https://quay.io/repository/redhat-developer/devconsole-operator)
 [![Docker Repository on Quay](https://quay.io/repository/redhat-developer/operator-registry/status "Docker Repository on Quay")](https://quay.io/repository/redhat-developer/operator-registry)
 
-DevConsole operator enables a developer-focused view in OpenShift 4.
-It provides a view switcher to transition between the traditional
-Kubernetes cluster administration console referred to as
-Administrator, to this new Developer perspective.
+## Overview
+The DevConsole operator enables a developer-focused view in the OpenShift 4 web console.
+It provides a view switcher to transition between the traditional Kubernetes cluster administration console referred to as Administrator, to a new Developer perspective.
 
-This new Developer perspective provides a high-level abstraction over
-Kubernetes and OpenShift primitives to allow developers to focus on
-their application development.
+This new Developer perspective provides a high-level of abstraction over Kubernetes and OpenShift primitives to allow developers to focus on their application development.
 
 ## Key Features
 
-The Developer perspective is still under active development.  These
-are the main features that are getting developed:
+The Developer perspective is under active development. These
+are the main features being developed:
 
-* Add - The place to create and build the application using one of this method:
+* **Add**: Use this page to create and build an application using one of the following methods:
 
-	- Importing source code from Git
-	- Deploying an existing image
-	- Browse a catalog to deploy or connect application services
-	- Deploy quick-starters or samples
+    - Import source code from Git
+    - Deploy an existing image
+    - Browse the catalog to deploy or connect application services
+    - Deploy quick-starts or samples
 
-* Topology - The landing page that shows application structure and
-  health in an easy-to-use diagram
-* Builds - Lists OpenShift BuildConfig resources for the selected
-  project
-* Pipelines - Lists Tekton Pipeline resources for the selected project
+* **Topology**: The landing page that shows the application structure and
+  health in an easy-to-use graphic representation.
+* **Builds**: This page lists the OpenShift BuildConfig resources for the selected project.
+* **Pipelines**: This page lists the Tekton Pipeline resources for the selected project.
+
+## Installing the latest console with the developer perspective as a non-admin user
```

Comment:
This is taken from `hack/install_devconsole/REAME.md`, right? So do we really need that README there?

Reply:
cc. @Avni-Sharma ^^ see the above comment.

Reply:
As discussed Baiju, having this in the main readme will put the content upfront and end user need not dig further for this information. @Avni-Sharma please let us know your thoughts.

Reply:
Would we also inform them how to see this perspective i.e by directing them to the install dir and then to the script?